CHANGELOG.md

```diff
@@ -0,0 +1,58 @@
+# Apium Changelog
+
+All notable changes to Apium will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic
+Versioning](https://semver.org/spec/v2.0.0.html).
+
+<!-- Template - Please preserve this order of sections
+## [Unreleased] - Brief description
+[Unreleased]: https://github.com/CDRH/api/compare/v#.#.#...dev
+
+### Fixed
+
+### Added
+
+### Changed
+
+### Removed
+
+### Migration
+
+### Deprecated
+
+### Contributors
+-->
+
+## [Unreleased]
+
+### Fixed
+
+### Added
+
+### Changed
+
+### Migration
+
```

Comment:
Don't think we want these lines in the release that's merged to `master`

Also, you have the repo name above as Apium, but the readme says Henbit. Which is it? :grimacing:

Reply:
ARGH that darn readme.  I'm pretty sure that Henbit is the name Karin picked when she was trying to centralize the documentation for the api, orchid, datura, etc.  It should not be a thing anymore.

And okay, I can erase those lines.  Makes sense that we wouldn't have those AND the blank commented out versions, but for some reason I was thinking we needed them!